accepted/00000-rbac.md

```diff
@@ -0,0 +1,285 @@
+It does this by matching the requested URI with the `endpoint` field of allowed endpoints within any associated namespace before passing the request along the chain.
+
+Additionally, the filter injects details of the relevant access rule, such as the namespace and the access mode, into the request context, allowing the controller processing the endpoint to access this information.
+This enables the controller to serve conditional content based on each user's access rules, such as hiding unauthorized sections of the page. New ACL methods can also be used to check access to arbitrary namespaces for similar purposes.
+
+The filter controls access to the following types of endpoints:
+
+- Struts pages
+- Spark web UI endpoints (pages and internal API)
+- Spark public API endpoints (JSON over HTTP API)
+- Saltboot endpoints
+- Ajax endpoints
+- Websocket endpoints
+
+By default, any requested URI is denied access unless a match is found.
+
+### XML-RPC API
+
+Access to XML-RPC API methods is managed through the `BaseHandler` class, the base class for all XML-RPC handlers.
+Access is granted if the requested method's name and handler match the `class_method` field of an allowed endpoint.
+
+XML-RPC and HTTP APIs share the same endpoint entries in the database. However, while HTTP requests are validated against the `endpoint` field, XML-RPC requests are validated against the `class_method` field.
+
+### Performance
+
+Access control checks query the database on every request.
+To mitigate performance impact, a user's access rules can be preloaded into memory and stored in the user's session data during login, enabling faster access control validation.
+
+
+## 4. Management Interface
+
+A UI catalog is proposed for access control administration, offering a clear and user-friendly view of all namespaces along with their descriptions.
+
+Additionally, administrators can manage access through a new API namespace called `access`, which provides methods to:
+
+- Grant or revoke access to a namespace for a user or access group
+- Add or remove a user from an access group
+- Check if a user or group has access to a specific namespace
+- List all available namespaces for a particular feature
+
+
+## 5. Development Resources
+
+After the initial development and mapping of all existing endpoints to the new structure, ongoing effort is required to keep endpoint mappings and the namespace structure up-to-date. To reduce RBAC overhead when implementing new features, the following additional resources are proposed:
+
+- An extensive Wiki guide on adding necessary endpoints with proper namespace organization
+- Stored procedures and scripts to easily grant/revoke user access for development
+- CI checks to detect any unmapped endpoints when introducing new features
+- A failsafe mechanism to prevent access to unmapped endpoints
+- Startup checks in the Uyuni Server to log any misconfigurations within a specific deployment
+
+# Drawbacks
+[drawbacks]: #drawbacks
```

Comment:
Spacing throughout the document is inconsistent, can you parse it through https://github.com/DavidAnson/markdownlint ?